### ISSUE TITLE
Accessibility in Blacklight/Hydra

### ISSUE THREAD

**Issue by richardGreen** [pbinkley/HC2015-program#84]
Suggested by: Program Committee
Presenter: Mike Tribone with panel
Format: Panel
